### ISSUE TITLE
Is it possible or can you make it possible to used the pre-trained model from link below?

### ISSUE THREAD

**Issue by arsalan993** [meijieru/crnn.pytorch#98]
Recently TextBoxes ++ https://github.com/MhLiao/TextBoxes_plusplus , uploaded their paper and in their implementation for text recognition part they are using CRNN and they also referred to CRNN implementation by https://github.com/bgshih/crnn , 
Now since there is issue with main CRNN software, it is not been able to run on ubuntu 17.04. 
You implementation is working well.. so can i used the pre-trained model referred in TextBoxes++ repository and use it in your implementation. 
Since he has trained the model using "SynthText " and also on "ICDAR 2015 Incidental Text" dataset.  therefore i believe it is going to have much more accuracy. 
Kindly somehow make it possible.

**Comment by meijieru** [comment]
This repository provide the tool for convert it.

**Comment by arsalan993** [comment]
Can you please guide me in how to use that tool in "tool" folder..  What should be the arguments

**Comment by meijieru** [comment]
dump/load model with tools in the dir `tool`
